<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ktodo</class>
 <widget class="QMainWindow" name="Ktodo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ktodo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QPushButton" name="pushButton_firstWidget">
          <property name="text">
           <string>我的首页</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton_secondWidget">
            <property name="text">
             <string>我的待办</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_todoCount">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>（0）</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>分组：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tableView">
          <property name="textElideMode">
           <enum>Qt::ElideRight</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButton_addGroup">
            <property name="text">
             <string>添加分组</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_addGroup"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout"/>
        </item>
       </layout>
      </widget>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QSplitter" name="splitter">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QWidget" name="layoutWidget">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QTableView" name="tableView_actTodo"/>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_done">
                 <property name="text">
                  <string>显示已完成：</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTableView" name="tableView_actDone"/>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QPushButton" name="pushButton_addAct">
                   <property name="text">
                    <string>新建代办</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_addAct"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QTreeView" name="treeView"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QStackedWidget" name="stackedWidget_2">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_5">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>点击左侧待办，查看详情</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_6">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="1" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>优先级：</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="lineEdit_description"/>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>备注：</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>描述：</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>事项：</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="lineEdit_location"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>提醒时间：</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="comboBox_priority">
               <property name="editable">
                <bool>true</bool>
               </property>
               <item>
                <property name="text">
                 <string>0</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>3</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QDateEdit" name="dateEdit_duetime"/>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>地点：</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDateTimeEdit" name="dateTimeEdit_remtime"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>到期时间：</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QPlainTextEdit" name="plainTextEdit_note"/>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_activity"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>确认修改</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>863</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>排序方式</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_orderDue"/>
    <addaction name="action_orderCreate"/>
    <addaction name="action_orderPriority"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>刷新</string>
    </property>
    <addaction name="action_refresh"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_orderDue">
   <property name="text">
    <string>到期时间排序</string>
   </property>
  </action>
  <action name="action_orderCreate">
   <property name="text">
    <string>创建时间排序</string>
   </property>
  </action>
  <action name="action_orderPriority">
   <property name="text">
    <string>优先级排序</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>添加分组</string>
   </property>
  </action>
  <action name="action_deleteGroup">
   <property name="text">
    <string>删除分组</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_refresh">
   <property name="text">
    <string>我的首页刷新</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
